reana_job_controller/config.py

```diff
@@ -80,3 +84,9 @@
 """Kerberos configMap name. Must be the same as in
 reana_cluster/backends/kubernetes/templates/configmaps/kerberos.yaml.
 """
+
+SLURM_CLUSTER_HOSTNAME = os.getenv('SLURM_HOSTNAME', '188.185.70.122')
```

Comment:
Do we know the real hostname and port for the CERN Slurm?
- Either set it to that value
- Or just to some default which is going to fail for sure so we notice, such as `localhost` for the hostname and `22` for the port (this pod doesn't have anything running on port `22`).

This is because, otherwise, we could risk that it connects to a different machine (this IP could stop being "ours", and almost for sure with `sshd` running) and it could be difficult to debug.

Reply:
Sure, we can set it latter to the real CERN hostname and port. I left these values for testing now.